### ISSUE TITLE
Misleading status message when using `fit` or `fdescribe`

### ISSUE THREAD

**Issue by PhilippMeissner** [karma-runner/karma-jasmine#202]
The output states to have run all available tests, but we were running into "silently failing" tests as no logs are present to explicitly state the number of tests run.

This is the output we have when running all our tests without any `fit` or `fdescribe` specs:
> Executed 473 of 473 SUCCESS (0.443 secs / 0 secs)

Now, using a single `fit` on one of the specs:
> Executed 473 of 473 SUCCESS (0.454 secs / 0 secs)

If I let the `fit`ed test fail explicitly, this is the output
> Executed 473 of 473 (1 FAILED) (0.397 secs / 0 secs)

Having `fdescribe` causes the very same output as above (Both scenarios, with and without failing tests).

We would except an output similar to when `xit` or `xdescribe` is present:
(with 1 `xit` in place)
> Executed 472 of 473 (skipped 1) SUCCESS (0.49 secs / 0 secs)

```
jasmine@3.1.0
jasmine-core@3.1.0
jasmin-local-storage@1.0.0
karma-jasmine@1.1.1
```

Hit me up if there's additional information required :)

**Comment by michAtEl** [comment]
I think I have the same problem. I made a small repository to reproduce the problem:
https://github.com/michAtEl/karma-bug
I have two specs. Spec `fit('hello returns hello', function () {` is focused with fit. Because of this, the other spec should not run. If it runs, it should fail because of `expect(2).toEqual(1);`
Output of karma:

> 31 03 2018 11:05:25.296:WARN [karma]: No captured browser, open http://localhost:9876/
31 03 2018 11:05:25.307:INFO [karma]: Karma v2.0.0 server started at http://0.0.0.0:9876/
31 03 2018 11:05:25.307:INFO [launcher]: Launching browser Chrome with unlimited concurrency
31 03 2018 11:05:25.316:INFO [launcher]: Starting browser Chrome
31 03 2018 11:05:27.128:INFO [Chrome 65.0.3325 (Windows 10.0.0)]: Connected on socket MffgRawUJx5m_oozAAAA with id 89040756
Chrome 65.0.3325 (Windows 10.0.0): Executed 2 of 2 SUCCESS (0.024 secs / 0 secs)

I would like to see something like this:
> Executed 1 of 2 SUCCESS (skipped 1) (0.024 secs / 0 secs)


Steps to reproduce:
- clone https://github.com/michAtEl/karma-bug
- npm install
- karma start karma.conf

I'm running:
- win 10
- karma-cli version 1.0.1
- chrome version 65.0.3325.181

This issue might be related to
https://github.com/karma-runner/karma/issues/2587

**Comment by michAtEl** [comment]
After a downgrade to the latest Version of 2.x ( `npm install --save jasmine-core@2.99.1` ) the output is ok:
```
karma start karma.conf
02 04 2018 10:51:05.481:WARN [karma]: No captured browser, open http://localhost:9876/
02 04 2018 10:51:05.492:INFO [karma]: Karma v2.0.0 server started at http://0.0.0.0:9876/
02 04 2018 10:51:05.493:INFO [launcher]: Launching browser Chrome with unlimited concurrency
02 04 2018 10:51:05.503:INFO [launcher]: Starting browser Chrome
02 04 2018 10:51:07.368:INFO [Chrome 65.0.3325 (Windows 10.0.0)]: Connected on socket n8zN-9nwxlg8Nca9AAAA with id 40420476
Chrome 65.0.3325 (Windows 10.0.0) ERROR: 'DEPRECATION: fit and fdescribe will cause your suite to report an 'incomplete' status in Jasmine 3.0'

Chrome 65.0.3325 (Windows 10.0.0): Executed 1 of 2 (skipped 1) SUCCESS (0.014 secs / 0 secs)

```

**Comment by PhilippMeissner** [comment]
> Chrome 65.0.3325 (Windows 10.0.0) ERROR: 'DEPRECATION: fit and fdescribe will cause your suite to report an 'incomplete' status in Jasmine 3.0'

Makes me wonder - We're using 3.1.0 and none of the above do report any different status (Which would be highly appreciated).

**Comment by PhilippMeissner** [comment]
It appears this error has been reported to `jasmine` already, but responsibility for the bug is being pushed back towards `karma-jasmine`.

https://github.com/jasmine/jasmine/issues/1532

**Comment by fetis** [comment]
There's a PR for it https://github.com/karma-runner/karma-jasmine/pull/192 and it was even merged, but I was unable to find the new version of it. In my opinion it's critical bug, so I would like to have fix ASAP.

**Comment by the-spyke** [comment]
@johnjbarton Any ETA on a release? This bug is stopping us from using Jasmine 3, and the fix is already merged.

**Comment by yoni-tock** [comment]
Posting to follow

**Comment by saif-adnan** [comment]
Posting to follow

**Comment by ObsidianSky** [comment]
Posting to follow

**Comment by christopherthielen** [comment]
Please do not "post to follow".  Click the Subscribe button on the right sidebar instead.

<img width="268" alt="screen shot 2018-04-25 at 5 36 23 pm" src="https://user-images.githubusercontent.com/2053478/39279674-4c64798a-48af-11e8-81ff-78ca76249cba.png">

Sorry for adding to the noise.

**Comment by PhilippMeissner** [comment]
> Please do not "post to follow". Click the Subscribe button on the right sidebar instead.+
Maybe people keep posting and add "noise" so that @dignifiedquire or other users with write-access do read and bump the version :)

**Comment by whyboris** [comment]
New release created 🎉 https://github.com/karma-runner/karma-jasmine/releases/tag/v1.1.2 🎉 
Works great, fixes this issue 🎖

**Comment by johnjbarton** [comment]
Fixed by #192
